metrics-dropwizard/src/main/java/com/mesosphere/usi/metrics/dropwizard/GarbageCollectorMetricSet.java

```diff
@@ -0,0 +1,52 @@
+package com.mesosphere.usi.metrics.dropwizard;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricSet;
+
+import java.lang.management.GarbageCollectorMXBean;
+import java.lang.management.ManagementFactory;
+import java.util.*;
+import java.util.regex.Pattern;
+
+import static com.codahale.metrics.MetricRegistry.name;
+
+/**
+ * A set of gauges for the counts and elapsed times of garbage collections.
+ *
+ * This is a copy of GarbageCollectorMetricSet from Dropwizard metrics, slightly adjusted to support USI metric name
+ * conventions.
+ */
+public class GarbageCollectorMetricSet implements MetricSet {
```

Comment:
Is this from Ivan? Why is it in Java?

Reply:
As the comment there says: 
```
 This is a copy of GarbageCollectorMetricSet from Dropwizard metrics, slightly adjusted to support USI metric name conventions.
```
Same with other collectors. And the reporters exits for the same reason - it's impossible to find one that doesn't screw up with metric names.